content/rest/guides/scripting-with-the-rest-api-and-net.md

```diff
@@ -0,0 +1,378 @@
+    {
+        var appId = "your_app_id"; 
+        var privateKey = "your_private_key"; 
+
+        var github = new GitHubClient(new ProductHeaderValue("YourApp"))
+        {
+            Credentials = new Credentials(GetJwtToken(appId, privateKey), AuthenticationType.Bearer)
+        };
+
+        var owner = "github";
+        var repo = "docs";
+        var pullNumber = 191;
+
+        var changedFiles = await GetChangedFiles(github, owner, repo, pullNumber);
+
+        var filePathRegex = new Regex(@"/data/", RegexOptions.IgnoreCase);
+        if (changedFiles.Any(fileName => filePathRegex.IsMatch(fileName)))
+        {
+            var commentUrl = await CommentIfDataFilesChanged(github, owner, repo, pullNumber);
+            Console.WriteLine($"Comment added at: {commentUrl}");
+        }
+    }
+
+    static string GetJwtToken(string appId, string privateKey)
+    {
+        var generator = new GitHubJwtFactory(
+            new StringPrivateKeySource(privateKey),
+            new GitHubJwtFactoryOptions
+            {
+                AppIntegrationId = int.Parse(appId), 
+                ExpirationSeconds = 600 
+            }
+        );
+
+        return generator.CreateEncodedJwtToken();
+    }
+
+    static async Task<IReadOnlyList<string>> GetChangedFiles(GitHubClient github, string owner, string repo, int pullNumber)
+    {
+        var filesChanged = new List<string>();
+
+        try
+        {
+            var iterator = await github.Repository.PullRequest.Files(owner, repo, pullNumber);
+
+            await foreach (var fileData in iterator)
+            {
+                filesChanged.Add(fileData.Filename);
+            }
+        }
+        catch (ApiException ex)
+        {
+            Console.Error.WriteLine($"Error! Status: {ex.StatusCode}. Message: {ex.Message}");
+        }
+
+        return filesChanged;
+    }
+
+    static async Task<string> CommentIfDataFilesChanged(GitHubClient github, string owner, string repo, int pullNumber)
+    {
+        var changedFiles = await GetChangedFiles(github, owner, repo, pullNumber);
+
+        var filePathRegex = new Regex(@"/data/", RegexOptions.IgnoreCase);
+        if (!changedFiles.Any(fileName => filePathRegex.IsMatch(fileName)))
+        {
+            return null;
+        }
+
+        try
+        {
+            var comment = await github.Issue.Comment.Create(owner, repo, pullNumber,
+                $"It looks like you changed a data file. These files are auto-generated. \n\nYou must revert any changes to data files before your pull request will be reviewed.");
+
+            return comment.HtmlUrl;
+        }
+        catch (ApiException ex)
+        {
+            Console.Error.WriteLine($"Error! Status: {ex.StatusCode}. Message: {ex.Message}");
+            return null;
+        }
+    }
+}
+```
+
+## Next steps
+
+- To learn more about Octokit.net see [the .NET Octokit SDK documentation](https://github.com/octokit/octokit.net).
```

Comment:
This links to the repository, not the documentation. Please fix either the link or the description.